src/test/java/com/ddd/moodof/acceptance/TagAcceptanceTest.java

```diff
@@ -0,0 +1,92 @@
+package com.ddd.moodof.acceptance;
+
+import com.ddd.moodof.application.dto.TagDTO;
+import com.ddd.moodof.domain.model.user.User;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.web.util.UriComponentsBuilder;
+import java.util.List;
+import static com.ddd.moodof.adapter.presentation.TagController.API_TAG;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertAll;
+
+public class TagAcceptanceTest extends AcceptanceTest{
+    private Long userId;
+
+    @Override
+    @BeforeEach
+    void setUp() {
+        super.setUp();
+        User user = signUp();
+        userId = user.getId();
+    }
+
+    @Test
+    public void 유저아이디_태그_전체_조회() throws Exception {
+        // given
+        태그_생성(userId, "tag1");
+        태그_생성(userId, "tag2");
+        태그_생성(userId, "tag3");
+        TagDTO.TagResponse tagResponse = 태그_생성(userId, "tag4");
+
+        // when
+        String uri =API_TAG;
+        List<TagDTO.TagResponse> response = getListWithLogin(uri, TagDTO.TagResponse.class, userId);
+
+        // then
+        assertAll(
+                () -> assertThat(response.size()).isEqualTo(4),
+                () -> assertThat(response.get(0).getName()).isEqualTo("tag1")
+        );
+    }
+
+    @Test
+    public void 태그를_생성한다() throws Exception {
+
+        // given
+
+        // when
+        TagDTO.TagResponse response = 태그_생성(userId, "name1");
+
+        // then
+        assertAll(
+                () -> assertThat(response.getId()).isNotNull(),
+                () -> assertThat(response.getUserId()).isEqualTo(userId),
+                () -> assertThat(response.getName()).isEqualTo("name1"),
+                () -> assertThat(response.getCreatedDate()).isNotNull(),
+                () -> assertThat(response.getLastModifiedDate()).isNotNull()
+        );
+    }
+    @Test
+    public void 태그를_삭제한다() throws Exception {
+        // given
+        태그_생성(userId, "name1");
+        TagDTO.TagResponse response = 태그_생성(userId, "name2");
+
+        // when
+        태그_리스트_출력(userId);
+        deleteWithLogin(API_TAG, response.getId(), userId);
+        태그_리스트_출력(userId);
+    }
+    @Test
+    public void 태그_수정() throws Exception {
+        // given
+        TagDTO.TagResponse createList = 태그_생성(userId, "name1");
+
+        // when
+        TagDTO.TagResponse updateList = 태그_수정(userId, createList.getId(),"name2");
+
+        // then
+        assertThat(createList.getName()).isNotEqualTo(updateList.getName());
```

Comment:
```suggestion
        assertThat(updateList.getName()).isEqualTo("name2");
```

는 어떠신가요? 업데이트 요청을 했더니, 요청한대로 이름이 변경되어 응답이 나왔더라~ 정도로 생각할 수 있을것 같아요

Reply:
요청흐름상 업데이트 기준으로 테스트하는것이 좋긴하겠네요! 생성기준으로 테스트할 필요는 없어보입니다 수정해놓겠습니다!!